test/api/base.js

```diff
@@ -0,0 +1,82 @@
+const assert = require("assert");
+const ripe = require("../..");
+
+describe("PersonAPI", function() {
+    it("should be able to construct a new person", async () => {
+        const api = new ripe.API();

```

Comment:
As the API is always the same, maybe move `const api = new ripe.API();` to line 5.
After`describe("PersonAPI", function() {` and before `it("should be able to construct a new person", async () => {`